jmix-jmxconsole/jmxconsole/src/main/resources/io/jmix/jmxconsole/mbean-detail-view.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright 2022 Haulmont.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<view xmlns="http://jmix.io/schema/flowui/view"
+      title="msg://mbeanDetailView.title">
+    <data>
+        <instance id="mbeanDc" class="io.jmix.jmxconsole.model.ManagedBeanInfo">
+            <collection id="attrDc" property="attributes"/>
+        </instance>
+    </data>
+    <facets>
+        <dataLoadCoordinator auto="true"/>
+    </facets>
+    <actions>
+        <action id="close" type="detail_discard" text="msg://actions.Close"/>
+    </actions>
+    <layout>
+        <span id="mbeanName"/>
+        <span id="mbeanClassName"/>
+        <span id="mbeanDescription"/>
+        <hbox classNames="buttons-panel">
+            <button action="attributesDataGrid.refresh"/>
+            <button action="attributesDataGrid.edit"/>
+        </hbox>
+        <dataGrid id="attributesDataGrid" dataContainer="attrDc" height="40em" width="100%">
```

Comment:
1. don't you think that it's tooo much? 
2. For some reason the size doesn't work. Try to open mbean with and without attributes